openai.audio/Ballerina.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 license = ["Apache-2.0"]
-keywords = ["AI/OpenAI", "Cost/Paid", "speech-to-text", "whisper"]
+keywords = ["AI/Audio", "Cost/Paid", "speech-to-text", "whisper", "Vendor/OpenAI"]
```

Comment:
shouldn't whisper be "Whisper"?

Reply:
Addressed in https://github.com/hevayo/ballerina-openai-client/pull/22/commits/7d8940561515b3965ab48cc3c02f90e5a83a8627